providers/gitlab/provider.go

```diff
@@ -0,0 +1,73 @@
+package gitlab
+
+import (
+	"context"
+	"strings"
+
+	"github.com/stamm/dep_radar"
+	"github.com/xanzy/go-gitlab"
+)
+
+var (
+	_ dep_radar.ITagGetter = &Provider{}
+	_ dep_radar.IProvider  = &Provider{}
+)
+
+// Provider for gitlab
+type Provider struct {
```

Comment:
what about getting repos from gitlab?

Reply:
It's true, but it's better. You can grub all the apps from gitlab and create table with all apps